source/includes/gcp/_target_proxies.md

```diff
@@ -64,14 +65,18 @@ curl -X GET \
 ```json
 {
     "data": {
-        "creationTimestamp": "2019-10-01T08:52:41.664-07:00",
-        "id": "2137054791002409126",
-        "kind": "compute#targetHttpProxy",
-        "name": "targetProxyName",
-        "selfLink": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/targetHttpProxies/targetProxyName",
-        "urlMap": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/urlMaps/urlMapName",
-        "type": "HTTP",
-        "shortUrlMap": "urlMapName"
+         "creationTimestamp": "2019-10-01T08:52:41.664-07:00",
+         "id": "2137054791002409126",
+         "kind": "compute#targetHttpsProxy",
+         "name": "targetProxyName",
+         "selfLink": "https://www.googleapis.com/compute/v1/projects/testAreaName/global/targetHttpsProxies/targetProxyName",
```

Comment:
Please match the selfLink with the url we usually use for docs: `"https://cloudmc_endpoint/v1/services/gcp/test-area/global/targetHttpsProxies/targetProxyName`

Reply:
Same for the links below.

Reply:
b9321225e3e3dcdaff19b7cb6d0dbec589231eb3